roles/awx/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Is this file needed? As far as I can tell it's not used by Ansible.

Reply:
It is needed for generation of awx-operator manifest file.